model/src/main/java/com/epam/esm/model/parameters/CertificateSortType.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.esm.model.parameters;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum CertificateSortType {
+    SORT_BY_NAME(CertificateParameter.CERTIFICATE_NAME),
+    SORT_BY_CREATE_DATE(CertificateParameter.CREATE_DATE);
+
+    CertificateParameter type;
```

Comment:
not clear variable purpose, think the name should be `sortParameter`

Reply:
Will be fixed in the next PR